hbp_knowledge/proteostasis/kim2013.bel

```diff
@@ -114,117 +121,153 @@ act(p(FPLX:HSP90)) -- bp(GO:"protein refolding")
 a(CHEBI:ATP) reg act(p(HGNCGENEFAMILY:Chaperonins))
 a(CHEBI:ATP) reg act(p(FPLX:HSPA))
 a(CHEBI:ATP) reg act(p(FPLX:HSP90))
+UNSET Confidence
 
 SET Evidence = "ATP-independent chaperones,
 such as the small Hsps, may function as
 additional holdases that buffer aggregation."
+SET Confidence = "High"
 a(CHEBI:ATP) cnc act(p(FPLX:HSPB))
 act(p(FPLX:HSPB)) -| a(MESH:"Protein Aggregates")
+UNSET Confidence
 
 SET Evidence = "TRiC binds to
 nascent chains and cooperates with Hsp70
 in the cotranslational folding of multidomain
 proteins (78). Investigators have demonstrated
 a direct interaction between Hsp70 and TRiC
 (79)."
+SET Confidence = "High"
 complex(p(FPLX:"CCT_complex"),p(FPLX:HSPA)) -> bp(GO:"'de novo' protein folding")
+UNSET Confidence
 
 SET Evidence = "Accordingly, Hsp90 affects many key cellular
 processes, including cell cycle progression,
 steroid signaling, calcium signaling, protein
 trafﬁcking, protein secretion, the immune re-
 sponse, and the heat shock response (HSR) (45,
 48, 82)."
+SET Confidence = "High"
 p(FPLX:HSP90) -- bp(GO:"cell cycle")
 p(FPLX:HSP90) -- bp(GO:"steroid hormone mediated signaling pathway")
 p(FPLX:HSP90) -- bp(GO:"calcium-mediated signaling")
 p(FPLX:HSP90) -- bp(GO:"protein transport")
 p(FPLX:HSP90) -- bp(GO:"protein secretion")
 p(FPLX:HSP90) -- bp(GO:"immune response")
 p(FPLX:HSP90) -- bp(MESH:"Heat-Shock Response")
+UNSET Confidence
 
 SET Evidence = "Hsp70 chaperones are a ubiquitous class of
 proteins. They are involved in a wide range
 of protein quality control functions, including
 de novo protein folding, refolding of stress-
 denatured proteins, protein transport, mem-
 brane translocation, and protein degradation."
+SET Confidence = "High"
 act(p(FPLX:HSPA)) -- bp(GO:"'de novo' protein folding")
 act(p(FPLX:HSPA)) -- bp(GO:"protein refolding")
 act(p(FPLX:HSPA)) -- bp(GO:"intracellular protein transmembrane transport")
 act(p(FPLX:HSPA)) -- bp(GO:"protein catabolic process")
+UNSET Confidence
 
 SET Evidence = " Hsp40 ( J protein) and NEF
 cochaperones regulate the Hsp70 reaction cy-
 cle (38, 100)."
+#RAPGEF refernce?
```

Comment:
NEF: nucleotide exchange factor, should be the same as RAPGEF (Rap guanine nucleotide exchange factor)
but I'm not sure about this